web/src/components/StatusBars/ResponseBar.jsx

```diff
@@ -48,7 +48,7 @@ export default function ResponseBar(props) {
     );
 }
 ResponseBar.propTypes = {
-    successText: PropTypes.string.isRequired,
+    successText: PropTypes.string,
     response: PropTypes.instanceOf(Promise).isRequired,
 };
```

Comment:
Making this optional is useful for cases where you want to show errors without caring about successes